webpack.config.js

```diff
@@ -217,11 +223,23 @@ module.exports = (env, argv) => {
       new HtmlWebpackPlugin({
         filename: 'index.html',
         template: 'src/index.html',
+        favicon: 'src/public/favicon.ico',
+        publicPath: '.',
         templateParameters: {
           rollbarSnippet,
           ...packageJson.config
         }
       }),
+      ...(DEPLOY_PATH ? [new HtmlWebpackPlugin({
+        filename: 'index-top.html',
+        template: 'src/index.html',
+        favicon: 'src/public/favicon.ico',
+        publicPath: DEPLOY_PATH,
+        templateParameters: {
+          rollbarSnippet,
+          ...packageJson.config
+        }
```

Comment:
Seems like you could dry up the code a bit with something like:
```
      const baseHtmlPluginConfig = {
        template: 'src/index.html',
        favicon: 'src/public/favicon.ico',
        templateParameters: {
          rollbarSnippet,
          ...packageJson.config
        }
      };
```
and then...
```
      new HtmlWebpackPlugin({
        filename: 'index.html',
        publicPath: '.',
        ...baseHtmlPluginConfig
      }),
      ...(DEPLOY_PATH ? [new HtmlWebpackPlugin({
        filename: 'index-top.html',
        publicPath: DEPLOY_PATH,
        ...baseHtmlPluginConfig
      }
```

Reply:
👍 I've made that change.